controller/server/view/status_page.ml

```diff
@@ -0,0 +1,33 @@
+open Tyxml.Html
+
+let section title_str content =
+  section
+    [ h2 ~a:[ a_class [ "d-Subtitle" ] ] [ txt title_str ]
+    ; pre ~a: [ a_class [ "d-Preformatted" ] ]  [ txt content ]
+    ]
+
+type params =
+  { health: string
+  ; update: string
+  ; rauc: string
+  ; interfaces: string
+  }
+
+let html { health; update; rauc; interfaces } =
+  Page.html (
+    div
+      [ h1
+          ~a:[ a_class [ "d-Title" ] ]
+          [ txt "System Statu"
```

Comment:
I am uneasy when I see this, but it’s done on purpose. The _s_ comes just after because it’s a link.

![2021-03-11_08-47-42](https://user-images.githubusercontent.com/6768842/110753123-765c1580-8246-11eb-8d89-227a8763a256.png)

Reply:
Ah I didn't notice the trick  :smile: